test/metabase/api/permissions_test.clj

```diff
@@ -287,108 +264,46 @@
-           (assoc-in (perms/data-perms-graph)
-                     [:groups (u/the-id group) (mt/id) :data :schemas] {"PUBLIC" {db-id :all}}))
-          (is (= (set (for [template ["/data/db/%s/schema/PUBLIC/table/%s/"
-                                      "/query/db/%s/schema/PUBLIC/table/%s/"
-                                      "/db/%s/schema/PUBLIC/table/%s/"]]
-                        (format template (mt/id) db-id)))
-                 (set (mapv :object (t2/select Permissions :group_id (u/the-id group))))))
-          (mt/user-http-request
-           :crowberto :put 200 "permissions/graph"
-           (assoc-in (perms/data-perms-graph)
-                     [:groups (u/the-id group) (mt/id)]
-                     {:data {:native "none" :schemas "none"}}))
-          (is (= #{}
-                 (set (mapv :object (t2/select Permissions :group_id (u/the-id group)))))))))))
+    (let [table-id (mt/id :venues)]
+      (t2.with-temp/with-temp [PermissionsGroup group]
+        (mt/user-http-request
+         :crowberto :put 200 "permissions/graph"
+         (assoc-in (data-perms.graph/api-graph)
+                   [:groups (u/the-id group) (mt/id) :data :schemas] {"PUBLIC" {table-id :all}}))
+        (is (= #{:unrestricted}
+               (set (t2/select-fn-set :perm_value :model/DataPermissions
+                                      :group_id  (u/the-id group)
+                                      :table_id  table-id
+                                      :perm_type :perms/data-access))))
+        (mt/user-http-request
+         :crowberto :put 200 "permissions/graph"
+         (assoc-in (data-perms.graph/api-graph)
+                   [:groups (u/the-id group) (mt/id)]
+                   {:data {:native "none" :schemas "none"}}))
+        (is (= #{:no-self-service}
+               (set (t2/select-fn-set :perm_value :model/DataPermissions
+                                      :group_id  (u/the-id group)
+                                      :db_id     (mt/id)
+                                      :perm_type :perms/data-access))))
+        (is (= #{}
+               (set (t2/select-fn-set :perm_value :model/DataPermissions
+                                      :group_id  (u/the-id group)
+                                      :table_id  table-id
+                                      :perm_type :perms/data-access))))))))
 
 (deftest update-perms-graph-error-test
   (testing "PUT /api/permissions/graph"
     (testing "make sure an error is thrown if the :sandboxes key is included in an OSS request"
       (mt/with-premium-features #{}
         (is (= "Sandboxes is a paid feature not currently available to your instance. Please upgrade to use it. Learn more at metabase.com/upgrade/"
                (mt/user-http-request :crowberto :put 402 "permissions/graph"
-                                     (assoc (perms/data-perms-graph) :sandboxes [{:card_id 1}]))))))))
-
-(deftest update-execution-perms-graph-global-execute-permission-test
```

Comment:
Removing this test as it was failing, but execution perms are unused and can be removed (https://metaboat.slack.com/archives/C04DN5VRQM6/p1708023271031159)

Will delete the rest of the execution perms code in another PR when we're doing more general cleanup